runtime/near-vm-runner/src/tests/wasm_validation.rs

```diff
@@ -136,18 +136,13 @@ fn extension_saturating_float_to_int() {
             "#,
         )
         .protocol_features(&[
-            ProtocolFeature::SaturatingFloatToInt,
             ProtocolFeature::FixContractLoadingCost,
         ])
         .expects(&[
             expect![[r#"
                 VMOutcome: balance 4 storage_usage 12 return data None burnt gas 0 used gas 0
                 Err: PrepareError: Error happened while deserializing the module.
             "#]],
```

Comment:
Is this test really passing? That's odd. My intuition is that it should now be `Err: MethodNotFound` both before and after `FixContractLoadingCost` as saturating float to int operations are always unconditionally enabled.